<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>120</width>
      <height>80</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbTitleBG">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>0</y>
      <width>261</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(50, 48, 48);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbTime">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Time</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbBattery">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>80</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>battery</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbWifi">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>110</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>wifi</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainWindow_lbEht">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>140</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>eht</string>
    </property>
   </widget>
   <zorder>MainWindow_lbTitleBG</zorder>
   <zorder>stackedWidget</zorder>
   <zorder>MainWindow_lbTime</zorder>
   <zorder>MainWindow_lbUser</zorder>
   <zorder>MainWindow_lbBattery</zorder>
   <zorder>MainWindow_lbWifi</zorder>
   <zorder>MainWindow_lbEht</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
